tests/sensu-client-windows.pp

```diff
@@ -1,6 +1,7 @@
 # Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+$pkg_repo = 'https://repositories.sensuapp.org/msi'
 class { '::sensu':
-  version           => '0.29.0-11',
+  windows_pkg_url   => "${pkg_repo}/2012r2/sensu-0.29.0-11-x64.msi",
```

Comment:
this is great for testing, though we should use the default of just specifying the version and see that the module does the right thing and figures out how to build the url appropriately.